crates/starknet-types-core/src/curve/affine_point.rs

```diff
@@ -18,6 +18,14 @@ impl AffinePoint {
         )?))
     }
 
+    pub const fn new_unchecked(x: Felt, y: Felt) -> AffinePoint {
```

Comment:
What’s the behavior if you send incorrect values ? Does it panic ? Can you add a small description over the function explaining that plz

Reply:
Just updated the documentation here. 

As per what is done here in the same context for Arkworks https://github.com/arkworks-rs/algebra/blob/bb663bc27e776af81e66c94d9016bde2dbba9968/ec/src/models/short_weierstrass/group.rs#L120, this method does not check if the point is on the curve or not.

As its name suggests, the method must be used with caution and full knowledge of the facts for very specific use cases.